--- a/internal/vdom/moon.pkg
+++ b/internal/vdom/moon.pkg
@@ -1,9 +1,11 @@
 import {
-  "XiLaiTL/moobile/dom",
-  "XiLaiTL/moobile/cmd",
-  "XiLaiTL/moobile/variant",
-  "XiLaiTL/moobile/js",
+  "XiLaiTL/moobile/style",
+  "XiLaiTL/moobile/vendor/rabbita/dom",
+  "XiLaiTL/moobile/vendor/rabbita/cmd",
+  "XiLaiTL/moobile/vendor/rabbita/variant",
+  "XiLaiTL/moobile/vendor/rabbita/js",
   "moonbitlang/core/ref",
+  "moonbitlang/core/immut/hashmap",
 }
 
 warnings = "-unused_package"
