--- a/internal/runtime/moon.pkg
+++ b/internal/runtime/moon.pkg
@@ -1,13 +1,14 @@
 import {
   "XiLaiTL/moobile/internal/vdom",
-  "XiLaiTL/moobile/dom",
-  "XiLaiTL/moobile/common",
-  "XiLaiTL/moobile/cmd",
-  "XiLaiTL/moobile/sub",
+  "XiLaiTL/moobile/vendor/rabbita/dom",
+  "XiLaiTL/moobile/vendor/rabbita/common",
+  "XiLaiTL/moobile/vendor/rabbita/cmd",
+  "XiLaiTL/moobile/vendor/rabbita/sub",
   "moonbitlang/core/queue",
   "moonbitlang/core/json",
-  "XiLaiTL/moobile/js",
+  "XiLaiTL/moobile/vendor/rabbita/js",
   "moonbitlang/async",
+  "moonbitlang/async/js_async",
   "XiLaiTL/moobile/internal/key",
   "XiLaiTL/moobile/internal/any",
   "XiLaiTL/moobile/internal/slotmap",
@@ -23,5 +24,6 @@
     "host_browser.mbt": [ "js" ],
     "host_hydration.mbt": [ "js" ],
     "host_ssr.mbt": [ "wasm", "native" ],
+    "react_host.mbt": [ "js" ],
   },
 )
